lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Converters/LMFeedConvertToFeedPost.swift

```diff
@@ -75,7 +75,16 @@ public struct LMFeedConvertToFeedPost {
     }
     
     public static func convertToFooterViewData(from data: LMFeedPostDataModel) -> LMFeedPostFooterView.ContentModel {
-        .init(likeCount: data.likeCount, commentCount: data.commentCount, isSaved: data.isSaved, isLiked: data.isLiked)
+        .init(
+            isSaved: data.isSaved,
+            isLiked: data.isLiked,
+            likeCount: data.likeCount,
+            commentCount: data.commentCount,
+            likeText: LMStringConstants.shared.likeVariable,
```

Comment:
Instead of sending two variable for likeText, can we check the count of likes and based on that send the text

```suggestion
            likeText: data.likeCount == 1 ? LMStringConstants.shared.likeVariable : LMStringConstants.shared.likeVariable.pluralize(),
```
Same goes for comment as well